csm/core/services/alerts.py

```diff
@@ -272,7 +279,7 @@ async def fetch_alert_for_support_bundle(self):
             False  #show_active
         )
         combined_alert_list.extend(resolved_alerts_list)
-        return [alert.to_primitive_filter_empty() for alert in combined_alert_list if not alert.module_type == const.IEM]
+        return [self.set_hostname_in_alert(alert.to_primitive_filter_empty) for alert in combined_alert_list if not alert.module_type == const.IEM]
```

Comment:
This line has syntax error. **to_primitive_filter_empty**

Reply:
done.